src/pages/Autolending/AutolendingWhen.vue

```diff
@@ -140,20 +146,30 @@ export default {
 					lendAfterDaysIdle
 				}
 			}
+			my {
+				autoDeposit {
+					donateAmount
+				}
+			}
 		}`,
 		preFetch: true,
 		result({ data }) {
 			this.isEnabled = !!_get(data, 'autolending.currentProfile.isEnabled');
-			const donationPercentage = _get(data, 'autolending.currentProfile.donationPercentage');
 			this.lendAfterDaysIdle = _get(data, 'autolending.currentProfile.lendAfterDaysIdle');
 			this.isChanged = !!_get(data, 'autolending.profileChanged');
+			const donationPercentage = _get(data, 'autolending.currentProfile.donationPercentage');
 			this.donation = _isFinite(donationPercentage) ? donationPercentage : 15;
```

Comment:
A cleaner solution instead of modifying the watches, we can set this.donation to be 0 in the results block if hasAutoDepositDonation

Reply:
I did that initially, then realized that would make the UI misleading; I think we should render whatever the server returns.